pkg/telemetry/rtt.go

```diff
@@ -0,0 +1,115 @@
+package telemetry
+
+import (
+	"math"
+	"time"
+
+	"github.com/pion/rtcp"
+)
+
+const (
+	NSUS                 = 1000
```

Comment:
maybe more descriptive name like `nanosInMicro` (that also does not sound great, but `NSUS` is very cryptic, I am assuming it is nano seconds in a micro second)

Reply:
done